corehq/apps/reports/views.py

```diff
@@ -1745,6 +1746,16 @@ def _error(msg):
             url = reverse('render_form_data', args=[domain, instance_id])
             return HttpResponseRedirect(url)
 
+        def get_non_parents(caseblocks):
+            return filter(lambda cb: cb.path == [], caseblocks)
+
+        def is_subcase(caseblock):
+            return 'create' in caseblock.caseblock
+
+        def get_matching_datum(caseblock, datums_):
+            datum = filter(lambda d: d.datum.id.endswith(caseblock.caseblock['path'][0]), datums_)
```

Comment:
I need to check this too, based on how the datum ID is built. Example values are:
```python
    caseblock.caseblock['path'] == ['subcase_0']
    d.datum.id == u'case_id_new_subcase_0'
```